contentreview-impl/impl/src/java/org/sakaiproject/contentreview/impl/advisors/DefaultSiteAdvisor.java

```diff
@@ -33,6 +34,12 @@ public boolean siteCanUseLTIReviewService(Site site) {
 		return true;
 	}
 	
+	@Override
+	public boolean siteCanUseLTIReviewServiceForAssignment(Site site, Date assignmentCreationDate)
```

Comment:
I'd be tempted to pass in the actual assignment object so that if someone wanted the advisor to check a property on the assignment they can do that without adding another method to the API.